master/static/migrations/20230504022913_add-cert-and-key-store.tx.up.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE cert_and_key_info (
+    id bigint unique NOT NULL,
```

Comment:
Since this isn't a standard autoincrement primary key, what about using `serial_number` as the column name?